frontend/components/CollateralTable.stories.js

```diff
@@ -7,11 +7,12 @@ const collaterals = generateFakeCollaterals();
 const common = {
     components: { CollateralTable },
     data: () => ({
-        collaterals,
+        collaterals: collaterals.collaterals,
+        onChainCollaterals: collaterals.onChain,
     }),
 };
 
 storiesOf('CollateralTable', module).add('Default', () => ({
     ...common,
-    template: '<CollateralTable :collaterals="collaterals" />',
+    template: '<CollateralTable :collaterals="collaterals" :on-chain-collaterals="onChainCollaterals" />',
```

Comment:
Why do you think the component should combine those two things? Couldn't it be done inside the store getter?